SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -748,10 +755,12 @@ void USpatialActorChannel::OnCreateEntityResponse(const Worker_CreateEntityRespo
 	UE_LOG(LogSpatialActorChannel, Verbose, TEXT("Created entity (%lld) for: %s."), Op.entity_id, *Actor->GetName());
 }
 
-void USpatialActorChannel::UpdateSpatialPosition()
+void USpatialActorChannel::UpdateSpatialPosition(bool bCheckLastUpdateTime)
```

Comment:
This is only ever called with a literal argument, and it the affect is limited to a check at the beginning of the function. Would it be better to split this into two methods, one of which calls the other if appropriate.

Reply:
Yep - good point - I'll look into it.